backend/pages/views.py

```
@@ -156,14 +156,16 @@ def edit_quarter(request, uuid=None):
         new_number = request.POST.get("number")
+        float_precision = request.POST.get("float_precision", 9)  # Default to 9 if not provided
 
         if not new_number:
             return redirect("manage_quarters")
 
         if quarter:
             quarter.number = new_number
+            quarter.float_precision = int(float_precision)
         else:
-            quarter = Quarter(number=new_number, user=request.user)
+            quarter = Quarter(number=new_number, float_precision=int(float_precision), user=request.user)
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)